### ISSUE TITLE
False positive consistency warning

### ISSUE THREAD

**Issue by Zert** [rrrene/credo#217]
### Precheck

I see consistency warning `There are spaces around operators most of the time, but not here.` for bit-syntax like this:

```
      <<_gw_size::little-integer-size(16),
        gw_flags::integer-size(8),
        conid::little-integer-size(16),  # <<---- here

        i1::integer-size(8),
        i2::integer-size(8),
        i3::integer-size(8),
        i4::integer-size(8),

        port::big-integer-size(16),  # <<---- here
        udp_data::binary()
      >>

```

### Environment

* Credo version (`mix credo -v`): `0.5.1`
* Erlang/Elixir version (`elixir -v`): `Erlang/OTP 19 [erts-8.1] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.3.4`

* Operating system: macOS Sierra 10.12.1

### What where you trying to do?

`mix credo -a`

### Expected outcome

No warnings here

### Actual outcome

`There are spaces around operators most of the time, but not here.`

**Comment by svoynow** [comment]
I think this is the same issue as https://github.com/rrrene/credo/issues/144

**Comment by rrrene** [comment]
@Zert Thx reporting! @svoynow is right, this is a duplicate.

I will keep this open as a reminder that we really need to fix this. :+1:

**Comment by devonestes** [comment]
Has this been addressed? Just looking at the tests for this check, and it seems to me like this should cover this case, right?

https://github.com/rrrene/credo/blob/master/test/credo/check/consistency/space_around_operators_test.exs#L38

**Comment by rrrene** [comment]
Closing this as it seems to be resolved. Please re-open if I am mistaken :+1:
